sql/src/main/java/io/crate/user/UnsupportedUserManager.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.user;
+
+import io.crate.analyze.user.Privilege;
+import io.crate.concurrent.CompletableFutures;
+import io.crate.exceptions.UnsupportedFeatureException;
+import io.crate.operation.user.ExceptionAuthorizedValidator;
+import io.crate.operation.user.StatementAuthorizedValidator;
+import io.crate.operation.user.User;
+import io.crate.operation.user.UserManager;
+
+import javax.annotation.Nullable;
+import java.util.Collection;
+import java.util.concurrent.CompletableFuture;
+
+public class UnsupportedUserManager implements UserManager {
```

Comment:
i think something like FallbackU... or StubU... would be better. pls also document the class